### ISSUE TITLE
Keyboard navigation does not work with 

### ISSUE THREAD

**Issue by brennanho** [tajo/react-range#157]
The arrow keys are not reversed when you change the direction of the slider.

**Comment by tajo** [comment]
repro with https://stackblitz.com/edit/react-range-edit?file=src%2FApp.tsx?
